qa-tests-backend/src/main/groovy/objects/Rsyslog.groovy

```diff
@@ -0,0 +1,56 @@
+package util
+
+import groovy.util.logging.Slf4j
+import objects.Deployment
+import objects.Service
+import orchestratormanager.OrchestratorMain
+
+@Slf4j
+class Rsyslog {
+    public static final Integer SYSLOG_PORT = 514
+    private Rsyslog() { }
+    private Service syslogSvc
+    private Deployment  deployment
+
+    static Rsyslog createRsyslog(OrchestratorMain orchestrator, String namespace) {
+        def deploymentName = "rsyslog"
+        def  rsyslog = new Rsyslog()
+
+        try {
+            rsyslog.deployment =
+                    new Deployment()
+                            .setNamespace(namespace)
+                            .setName(deploymentName)
+                            .setImage("rsyslog/syslog_appliance_alpine")
```

Comment:
We don't want our tests to directly pull from docker.io as it could get rate limited. So I moved this image to our own repository.

Please use `quay.io/rhacs-eng/qa:rsyslog_appliance_alpine` as the image instead.

And leave a comment above it:

```
// The original is at docker.io/rsyslog/syslog_appliance_alpine:8.36.0-3.7
// and https://github.com/rsyslog/rsyslog-docker
```